tf-m-tests/app/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 #-------------------------------------------------------------------------------
-# Copyright (c) 2020, Arm Limited. All rights reserved.
+# Copyright (c) 2020-2021, Arm Limited. All rights reserved.
```

Comment:
Same with this commit: 
> Synchronize with commit 402a298e2eb32dc1c72d991eacf300bc96e78416

Reply:
It's upstream tf-m-tests. https://review.trustedfirmware.org/plugins/gitiles/TF-M/tf-m-tests/+/402a298e2eb32dc1c72d991eacf300bc96e78416